### DISCUSSION TITLE
[VSA] Why are there so many different values to determine precision

### DISCUSSION THREAD

**Discussion by ponceta** [QGEP/QGEP#670] [Q&A] [answered]
There are actually many tables with different value lists for positional accuracy assessment :

ex :
vl_cover_positionnal_accuracy
vl_reach_elevation_determination
vl_horizontal_positioning

with each time different descriptive values.

When applying this to our daily job, it appears that what is needed is to know if you can dig directly or not (accurate or not). 

The precision of the measured point is stored in a gps database rather than in the operationnal database.

How do others feels about this?

What we think in Pully is that it would be better to store the link to the gps measurement and store the definite value (0.26 cm as example).

**Comment by max-trolliet** [comment]
I think it is directly from the VSA - SDEE model. But I share the concern. It could be simpler.

**Comment by sjib** [comment]
This comes from the VSA-SDEE model.
An accuracy for (x,y) and z has not the same precision. This also depends on the method. Therefore there are different value lists. But we introduced here absolute values to be able to compare objects that have been measured with different methods (GPS, digitizing, theodolite, measuring tape, ...)
Reach.Elevation.determination is originally only a value to define the drawing parameter (precise _______, imprecise - - - - line)

**Comment by sjib** [comment, marked as answer]
![deckel lagegenauigkeit](https://user-images.githubusercontent.com/1265865/34124936-01c6f292-e435-11e7-8e01-48d8793b20a3.png)
![haltung lagebestimmung](https://user-images.githubusercontent.com/1265865/34124937-01e339ca-e435-11e7-9de0-e193d8b15290.png)
![haltung hoehenbestimmung](https://user-images.githubusercontent.com/1265865/34125149-a3d9c906-e435-11e7-922f-e9226007b2e4.png)

**Comment by ponceta** [comment]
Thanks for the explanation.

As I understand, there are mainly two distinct accuracies 2D and 3D, depending on the acquisition method.

This enables to describe the accuracy of the measurement point, and therefore the overall position accuracy of the object.

IMHO we have here a conflict of interest between geomaticians and network operators because some want the qualitative value of the measurement accuracy (+-10cm, ...) and others want the qualitative value of the structure position accuracy (precise or not -> can or can't I dig there)

**Comment by ponceta** [comment]
Would it be at least possible to unify value lists for horizontal accuracy (2D) and unify value lists for vertical  accuracy (Z)?

Or are there some other limitations?

**Comment by sjib** [comment]
Measuring horizontally and vertically do not have the same accuracy range. Z values are less precise usually.

**Comment by m-kuhn** [comment]
I guess this is what @ponceta proposed

 * One list of accurracies for horizontal
 * One list of accurracies for vertical

@ponceta please correct me if I'm wrong

**Comment by ponceta** [comment]
@m-kuhn You're exactly right, the idea is to have planimetric precision and altimetric precision values, which is detailled enough.

**Comment by decheu** [comment]
There's also:
![image](https://user-images.githubusercontent.com/97091934/151196870-9fac8d6a-5aa6-4f55-89eb-eb8d2358b64b.png)
point_troncon.precision_altimetrique (or similar)

Therefore, I haven't understood the need/use of Haltung.Hoehenbestimmung.
Does anyone know why there is no Haltungspunkt.Lagegenauigkeit (nor Abwasserknoten.Lagegenauigkeit)? If there was, it should be possible to generate the display info (_____,------) dynamically and drop the Haltung.Lagebestimmung.
Like this (i. e. also without Haltung.Hoehenbestimmung), we would be a step further in implementing @ponceta's suggestion nicely phrased by @m-kuhn

Before doing this in QGEP, it would probably be wise to adapt VSA-DSS/SDEE first. I, therefore, posted the link to this discussion on [vsa.ch/wiki/gelbdrucke](https://vsa.ch/wiki/gelbdrucke).

**Comment by sjib** [comment]
These attributes have developed over time starting with 2D and then evolving towards 3D.

- At the beginning there was the Leitungskataster (SIA 405 Norm starting from 1998) that wanted to visualize with different line signatures whether a reach was accurate or not (Haltung.Lagebestimmung [genau, ungenau, unbekannt]
- Then there was an attribute that tried to go a little further in using the measurement method (such as digitizing, scanning of old maps, field measurements with different tools) - but this did not lead to good results as these methods could be used differently and was given up again.
- So to have more information how accurate measurement were it was looked at what excactly was measured in the field in a sewer network and this are usually the centers of the covers (Pickelloch), as this is the only clearly defined point you see -> Deckel.Lagegenauigkeit was introduced with absoulte values plus_minus_xxx to be indepentant of the measurement method.
- Then there was the adding of levels - and related to this one wanted an information about how accurate the calculated slopes are. This lead to Haltungspunkt.Hoehengenauigkeit, again wich an accuray. Reach_point (Haltungspunkt) was chosen as element, as the level of the reach point is also something physically exact you see in reality. 
- And to have the analog attribut in height also for how exact a reach is on the z-coordinate the attribut Haltung.Hoehenbestimmung was introduced (genau, ungenau, unbekannt) (pendant to Lagebestimmung)
- So you can use the accuracy attributes (Deckel.Lagegenauigkeit and Haltungspunkt.Hoehengenauigkeit) to automatically generate the Haltung.Lagebestimmung and Haltung.Hoehenbestimmung in matching what you see as genau (up to +/- 10 cm resp +/- 6cm) / and if it is more -> ungenau. When Haltungspunkt.Hoehenbestimmung is not precise (genau) on both sides of the reach then it will be also unprecise (ungenau).

Hope this helps to give a better picture.

**Comment by decheu** [comment]
Thanks for the history and explanations @sjib

Using Haltungspunkt.Hoehengenauigkeit of the two reach_points to calculate Haltung.Hoehenbestimmung sounds like the thing to do to me.

Good idea, to calculate also Haltung.Lagebestimmung @sjib. The problem is that the precision of the 2D coordinates of the cover is not good enough for the reach. For a typical setting with 1m-manholes, the reach axis can easily be off by 0.5 m compared to the middle of the cover (Pickelloch), because we mostly have excentric cones (upper part of the manhole below the cover) and the reach/canal might not be centred compared to the manhole. (Then of course, the reach_point is at the inner edge of the manhole, makes another 0.5m - this time on average - shift along the axis.)

Therefore, I suggest to do exactly the same for the 2D precision as for the height precision: i. e. use Haltungspunkt.Lagegenauigkeit of the two reach_points to calculate Haltung.Lagebestimmung.

Now, there is no Haltungspunkt.Lagegenauigkeit. So, could we add that? @sjib

**Reply by sjib** [reply]
@decheu Right now we assume that the Lagegenauigkeit of the Haltungspunkt is the same as the one from the Deckel as this is the reference point where you would may be measure a delta from. Usually you cannot place your measurement instrument on a Haltungspunkt, as it is 1 - 5 m deep down in the manhole. So you usually will measure / calculate some difference in delta x,y. And I do not think that someone has time and money to maintain different Lagegenauigkeiten in a wastewater_structure (on Deckel and on Haltungspunkt)? Same argument with z.

**Reply by decheu** [reply]
The thing is that 2D precision (Lagegenauigkeit) of the cover is not very interesting - as long as you find the right cover in the field. So, I would rather opt for maintaining a 2D precision of the reach_points.

Same for height precision: the height precision of the cover in itself is not very interesting for the sewer system (to know whether a manhole is overflowing in your hydraulic simulation, I suggest that a precision of +/- 10 cm is good enough --> if need be I can get that from a DTM). I agree with @sjib that the cover height precision is of interest if it's used to determine the altitude of the reach_points (via Abstiche / profondeurs).
But: Firstly, newly built manholes can (and imho should) be measured before the cone is put on top --> reach points + centre/geometry are acessible vertically from above to be measured directly (Btw: the fact that the centre of the manhole and not of the cover (Pickelloch) should me measured/used for 2D coordinates of the Abwasserknoten has been stated just last week at the VSA further education at Glattpark by JL). Secondly, nowadays with modern GNSS, you can incline the stick and this is compensated automatically. --> You can measure the actual reach_points even with an excentric cone in place, when your stick is longer than the manhole deep.

So, on the long run, I would use the reach_points precisions and drop the use of the cover precisions.

--> So (again): can we add Haltungspunkt.Lagegenauigkeit in VSA-DSS?
